### PR TITLE
React react2 week1

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: react/react2/week1/github-user-seracher/src/ReposContext.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import  {useEffect, useState, createContext} from 'react';
+
+ export const ReposContext = createContext()
+
+const ReposProvider = ({children}) => {
+    const [repos, setRepos] = useState([])
```
Comment (annamourasobral) [inline react/react2/week1/github-user-seracher/src/ReposContext.jsx]: You could move this state to the Context.js file and consume it here.

File: react/react2/week1/class-exercise/src/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (annamourasobral) [inline react/react2/week1/class-exercise/src/App.js:1]: these files got me confused

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hi, @27ManishaPatel 
great job!

File: react/react2/week1/github-user-seracher/src/ReposContext.jsx (hunk 1 above)
Comment (27ManishaPatel) [reply react/react2/week1/github-user-seracher/src/ReposContext.jsx]: Yes thats good idea 👍 💡  Thanks for highlighting this 🙈

File: react/react2/week1/class-exercise/src/App.js (hunk 2 above)
Comment (27ManishaPatel) [reply react/react2/week1/class-exercise/src/App.js:1]: May I ask why  and what ? its nice if you describe it and give some suggestions that how should I start it with. Thank you 😊

**Review by annamourasobral — APPROVED** [review]

**Pull request merged**